PCA_Comp_Analysis_Dredging.R

```diff
@@ -272,17 +290,121 @@ my.proj<- function(field=NA,pattern=NA) {
 
 
 ###############################################################
-# Main part of the script
+# (2) Main part of the script
 ###############################################################
-#select season, range of years, remove all stations that include an NA value
 
+###############################################################
+# (2.1) User defined parameters
+###############################################################
+
+# OET: Comments /Question
+# Since dredge process is done down below in the script 
+# for once season and one selected PC mode
+# I wonder if we add the user-defined variable to 
+# set the PC mode that we analyze here already
+# is it variable pcamode? 

```

Comment:
yes, pcamode is changed between 1,2,3 and 4 in code below. do you mean we can loop through these rather than change and run variable each time? or just moving it up to this code as one of the "user defined variables"
and season is changed between 'wet' and 'dry'

Reply:
Looping can be done, later, I suggest.